DataFormats/Detectors/ZDC/include/DataFormatsZDC/ChannelData.h

```diff
@@ -0,0 +1,44 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+#ifndef _ZDC_CHANNEL_DATA_H_
+#define _ZDC_CHANNEL_DATA_H_
+
+#include "ZDCBase/Constants.h"
+#include <array>
+#include <Rtypes.h>
+
+namespace o2
+{
+namespace zdc
+{
+
```

Comment:
could you add a small description of the class?

Reply:
ok